keen/explorer
# Fix 78: Deleting a saved query renders Explorer unusable

## What does this PR do? How does it affect users?

It fixes #78 by ensuring that when we create a new active explorer, we first create it *then* set it active using the appropriate action, `ExplorerActions.setActive`. Making an active explorer any other way causes issues.

## How should this be tested?

1) Create a saved query
2) Reload explorer
3) Delete the saved query
4) Select any other saved query + use the query builder and run queries
5) Ensure all actions in #4 work as expected

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:


## Related tickets?
#78
